tests/unit/preprocessor/_multimodel/test_multimodel.py

```diff
@@ -7,17 +7,19 @@
 from cf_units import Unit
 
 import tests
-from esmvalcore.preprocessor import multi_model_statistics
-from esmvalcore.preprocessor._multimodel import (_assemble_data,
-                                                 _compute_statistic,
-                                                 _get_time_slice, _plev_fix,
-                                                 _put_in_cube,
-                                                 _unify_time_coordinates)
+from esmvalcore.preprocessor._multimodel import (
```

Comment:
It would be good to restore the test that `multi_model_statistics` is imported from `esmvalcore.preprocessor` (instead of from `esmvalcore.preprocessor._multimodel`), because without this test it may be possible to remove the function from the public API without a test breaking

Reply:
Done!